__tests__/main.test.js

```diff
@@ -470,11 +493,86 @@ test("Includes pre-releases one with new tag", async () => {
       name: "1.1.0-0",
     },
   ]);
-  const tags = await calculateTags({
+  const { tags } = await calculateTags({
     token: "TOKEN",
     owner: "owner",
     repo: "repo",
     ref: "refs/tags/1.1.0-1",
   });
   expect(tags).toEqual(["1.1.0-1", "1.1.0", "1.1", "1"]);
 });
+
+test("Externally provided tags", async () => {
+  const tags = await calculateTagsFromList({
+    currentTag: "4.1.2-10",
+    tagList: [
+      "other",
+      "3.1.2",
+      "4.0.0",
+      "4.1.2",
+      "4.1.2-9",
+      "4.1.2-10invalid",
+      "4.1.2-10-invalid",
+      "4.2.0-0",
+      "5.2.2",
+      "5.2.2-2",
+    ],
+  });
+  expect(tags).toEqual(["4.1.2-10", "4.1.2", "4.1"]);
+});
```

Comment:
Is `tagList` including the current/new tag when fetching from a GitHub repo in a action job triggered by a tag push? It is right?

I currently expect the output to be the same no matter if the current/new tag is or isn't part of the existing tags - does that align with your expectations?

Reply:
Yes, I've added a second test